spacialgaze-plugins/misc.js

```diff
@@ -327,4 +327,52 @@ exports.commands = {
 			"|/text This user is currently offline. Your message will be delivered when they are next online.");
 	},
 	tellhelp: ["/tell [username], [message] - Send a message to an offline user that will be received when they log in."],
+
+	utoken: 'usetoken',
+	usetoken: function (target, room, user, connection, cmd, message) {
+		target = target.split(',');
+		if (target.length < 2) return this.parse('/help usetoken');
+		target[0] = toId(target[0]);
+		let msg = '';
+		if (['avatar', 'declare', 'icon', 'color', 'emote', 'title'].indexOf(target[0]) === -1) return this.parse('/help usetoken');
+		if (!user.tokens || !user.tokens[target[0]]) return this.errorReply('You need to buy this from the shop first.');
+
+		switch (target[0]) {
+		case 'avatar':
+			msg = '|html|<center>' + user.name + ' has redeemed a avatar token.<br/><img src="' + target[1] + '" alt="avatar"/><br/>';
+			msg += '<button class="button" name="send" value="/customavatar set ' + user.userid + ', ' + target[1] + '">Apply Avatar</button></center>';
+			delete user.tokens[target[0]];
+			return SG.messageSeniorStaff(msg);
+		case 'declare':
+			msg += '|html|<center>' + user.name + 'has redeemed a global declare token.<br/> Message: ' + target[1] + "<br/>";
+			msg += '<button class="button" name="send" value=/globaldeclare ' + target[1] + '">Globally Declare the Message</button></center>';
+			delete user.tokens[target[0]];
+			return SG.messageSeniorStaff(msg);
```

Comment:
Messages aren't being sent as parsed HTML.

`[13:41:20] ~SG Server: |html|<center>HoeenHerohas redeemed a global declare token.<br/> Message: Hello World!<br/><button class="button" name="send" value=/globaldeclare Hello World!">Globally Declare the Message</button></center>`

Also your missing a space between `usernamehas`
(this applies to all the cases in the switch statement, not just this one)